### DISCUSSION TITLE
Users logging in

### DISCUSSION THREAD

**Discussion by cyberkimh** [Security-Onion-Solutions/securityonion#4846] [Unsupported Versions] [answered]
Hello, does anyone know if there are any logs of when users have logged into the SOC or TheHive? I had a good look through the docs and through the folders on the management node but can't find anything.

**Comment by jertel** [comment, marked as answer]
`/opt/so/log/kratos/kratos.log` is a JSON formatted log showing SOC authentication activity. Not all log events have the user email but they do have the user ID which can be correlated to the user's email address.

**Reply by cyberkimh** [reply]
That's really helpful thank you.

**Reply by Stephen-Ridgway** [reply]
Possibly a really stupid question, but how do you correlate the user ID with the email address?
